packages/saber/vue-renderer/app/helpers/inject-config.js

```diff
@@ -1,24 +1,39 @@
-import { siteConfig, themeConfig } from 'saber/config'
+import { siteConfig, themeConfig, locales } from 'saber/config'
 
 export default ({ Vue }) => {
-  const configStore = Vue.observable({ siteConfig, themeConfig })
+  const store = Vue.observable({ siteConfig, themeConfig, locales })
+
+  const getMatchedConfig = (currentRoute, key) => {
+    const routes = Object.keys(store.locales)
+    let matchedRoute = '/'
+    for (const route of routes) {
+      if (route !== '/') {
+        if (currentRoute === route || currentRoute.indexOf(`${route}/`) === 0){
```

Comment:
```suggestion
        if (currentRoute === route || currentRoute.startsWith(`${route}/`)){
```

Reply:
that's not supported by IE 😄

Reply:
ah fuck :laughing: babel might be worth a shot going forward so that we don't have to think about :wastebasket:, there's also a vue build option coming that provides a babel build only for old browsers if i remember that correctly.